server/metrics/metrics.go

```diff
@@ -1016,6 +1016,14 @@ var (
 	//  )
 	// ```
 
+	FirecrackerExecDialDurationUsec = promauto.NewHistogram(prometheus.HistogramOpts{
```

Comment:
Sorry this feedback is a little late but what do you think about reusing the FirecrackerStageDurationUsec metric?